common/src/main/java/us/myles/ViaVersion/protocols/protocol1_13to1_12_2/data/NamedSoundRewriter.java

```diff
@@ -0,0 +1,152 @@
+        oldToNew.put("entity.parrot.imitate.illusion_illager", "entity.parrot.imitate.illusioner");
+        oldToNew.put("entity.parrot.imitate.magmacube", "entity.parrot.imitate.magma_cube");
+        oldToNew.put("entity.parrot.imitate.vindication_illager", "entity.parrot.imitate.vindicator");
+        oldToNew.put("entity.player.splash.highspeed", "entity.player.splash.high_speed");
+        oldToNew.put("entity.polar_bear.baby_ambient", "entity.polar_bear.ambient_baby");
+        oldToNew.put("entity.small_magmacube.death", "entity.magma_cube.death_small");
+        oldToNew.put("entity.small_magmacube.hurt", "entity.magma_cube.hurt_small");
+        oldToNew.put("entity.small_magmacube.squish", "entity.magma_cube.squish_small");
+        oldToNew.put("entity.small_slime.death", "entity.slime.death_small");
+        oldToNew.put("entity.small_slime.hurt", "entity.slime.hurt_small");
+        oldToNew.put("entity.small_slime.jump", "entity.slime.jump_small");
+        oldToNew.put("entity.small_slime.squish", "entity.slime.squish_small");
+        oldToNew.put("entity.snowman.ambient", "entity.snow_golem.ambient");
+        oldToNew.put("entity.snowman.death", "entity.snow_golem.death");
+        oldToNew.put("entity.snowman.hurt", "entity.snow_golem.hurt");
+        oldToNew.put("entity.snowman.shoot", "entity.snow_golem.shoot");
+        oldToNew.put("entity.vindication_illager.ambient", "entity.vindicator.ambient");
+        oldToNew.put("entity.vindication_illager.death", "entity.vindicator.death");
+        oldToNew.put("entity.vindication_illager.hurt", "entity.vindicator.hurt");
+        oldToNew.put("entity.zombie.attack_door_wood", "entity.zombie.attack_wooden_door");
+        oldToNew.put("entity.zombie.break_door_wood", "entity.zombie.break_wooden_door");
+        oldToNew.put("entity.zombie_pig.ambient", "entity.zombie_pigman.ambient");
+        oldToNew.put("entity.zombie_pig.angry", "entity.zombie_pigman.angry");
+        oldToNew.put("entity.zombie_pig.death", "entity.zombie_pigman.death");
+        oldToNew.put("entity.zombie_pig.hurt", "entity.zombie_pigman.hurt");
+        oldToNew.put("record.11", "music_disc.11");
+        oldToNew.put("record.13", "music_disc.13");
+        oldToNew.put("record.blocks", "music_disc.blocks");
+        oldToNew.put("record.cat", "music_disc.cat");
+        oldToNew.put("record.chirp", "music_disc.chirp");
+        oldToNew.put("record.far", "music_disc.far");
+        oldToNew.put("record.mall", "music_disc.mall");
+        oldToNew.put("record.mellohi", "music_disc.mellohi");
+        oldToNew.put("record.stal", "music_disc.stal");
+        oldToNew.put("record.strad", "music_disc.strad");
+        oldToNew.put("record.wait", "music_disc.wait");
+        oldToNew.put("record.ward", "music_disc.ward");
+
+    }
+
+    public static String getNewId(String old) {
+        String newId = oldToNew.get(old);
+        return newId != null ? newId : old;
```

Comment:
I would recommend doing `old.toLowerCase()`. I've noticed a few sounds in 1.13 have the same name, but with caps removed (for example `item.fireCharge.use` is now `item.firecharge.use`). Sending caps to the client will result in a disconnect.